eHealthAfrica/aether
# Aet 166 update module apis kong

#### What does this PR do?
Modifies modules to use `kong` as a proxy

#### How to test
Set `PROJECT_API_URL` to the specified domain of the project e.g: `api.aether.local`
Add ONLY `api.aether.local` to the /etc/hosts file
Run `docker-compose up --build kernel` or `docker-compose -f ./docker-compose.yml up --build`
You should be able to reach `kernel`, `odk` on `api.aether.local/aether/v1` and `api.aether.local/odk/v1` respectively 
Check that all operations can be performed with this URLS as was with `kernel.aether.local:8000` and `odk.aether.local:8443`

#### Related JIRA ticket
[AET-166](https://jira.ehealthafrica.org/browse/AET-166)
